horace_core/sqw/@axes_block/build_from_input_binning.m

```diff
@@ -87,6 +87,20 @@
             range = [range(1),range(3)];
         end
     end
+    if numel(range) == 2
+        if range(1)==range(2)  % some old file formats do not store proper img_range
+            % and store 0,0 for integration ranges. Here we try to mitigate
+            % this
+            av_pt = 0.5*(range(1)+range(2));
+            if abs(av_pt)<abs(SQWDnDBase.border_size)
+                range(1) = -abs(SQWDnDBase.border_size);
+                range(2) =  abs(SQWDnDBase.border_size);
+            else
+                range(1) = av_pt*(1-abs(SQWDnDBase.border_size));
+                range(2) = av_pt*(1+abs(SQWDnDBase.border_size));
+            end
```

Comment:
Now you have 2 cases: av_pt on the border - let it straddle a bit, and, av_pt off the border, push apart by a border width. May be worth explaining why only the 0 border is treated - are there any others?

Reply:
0-border is treated differently because multiplying by 0 gives 0, not a range. If av_pt is large, adding/subtracting ~`eps` to it do not make any difference. In this case, its better to treat border as relative

Reply:
as for the first notice -- that's probably correct. `av_pt == range(1)` if `range(1)==range(2)`